### ISSUE TITLE
UITableViewCell 这中[self.contentView  addSubView:self.titleLabbel]  与demo中 [self addSubView:self.titleLabbel]； 显示不一致

### ISSUE THREAD

**Issue by swlizhao** [tigerAndBull/TABAnimated#176]
### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.
#
我是 在UITabelViewCell底部添加一个背景UIImageView，其它控件添加在改UIImageView上，骨架屏没有把子组件显示出来，我的理解是 子组件已经显示，颜色冲突， 能否为每个子组件单独添加一个颜色，希望楼主可以回复
